prow/cmd/hmac/main.go

```diff
@@ -34,21 +34,22 @@ import (
 	"k8s.io/client-go/kubernetes"
 	"sigs.k8s.io/yaml"
 
-	"k8s.io/test-infra/pkg/flagutil"
 	"k8s.io/test-infra/prow/config"
 	"k8s.io/test-infra/prow/config/secret"
 	prowflagutil "k8s.io/test-infra/prow/flagutil"
 	"k8s.io/test-infra/prow/ghhook"
 	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/kube"
 	"k8s.io/test-infra/prow/logrusutil"
 )
 
 type options struct {
-	configPath string
+	kubeconfig    string
+	kubeconfigCtx string
+	configPath    string
 
-	dryRun     bool
-	github     prowflagutil.GitHubOptions
-	kubernetes prowflagutil.KubernetesOptions
```

Comment:
Please continue to use `flagutil.KubernetesOptions` rather than adding your own  `--kubeconfig` flag. That will avoid code duplication and allow the use of `--buildcluster` instead of `--kubeconfig`.

Reply:
Done.